tests/jerry/es2015/proxy_own_keys.js

```diff
@@ -12,7 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// TODO: Update these tests when the internal routine has been implemented
+// Copyright 2015 the V8 project authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
Cannot we do similar tests instead of copying them?

Reply:
I think that would be the same intellectual product. Also the licenses are compatible and several tests are already included from V8. I'd leave as it.